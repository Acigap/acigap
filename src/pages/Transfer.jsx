import React from 'react';

function Transfer() {
  return (
    <div className="space-y-6">
      <h1 className="text-2xl font-semibold text-[var(--neutral-12)]">โอนเงิน</h1>
      <div className="bg-[var(--neutral-1)] p-6 rounded-lg border border-[var(--border)]">
        <form className="space-y-4">
          <div>
            <label className="block text-sm font-medium text-[var(--neutral-11)] mb-1">
              บัญชีปลายทาง
            </label>
            <input
              type="text"
              className="w-full p-2 rounded border border-[var(--border)] bg-[var(--neutral-2)]"
              placeholder="หมายเลขบัญชี"
            />
          </div>
          <div>
            <label className="block text-sm font-medium text-[var(--neutral-11)] mb-1">
              จำนวนเงิน
            </label>
            <input
              type="number"
              className="w-full p-2 rounded border border-[var(--border)] bg-[var(--neutral-2)]"
              placeholder="0.00"
            />
          </div>
          <button
            type="submit"
            className="w-full bg-[var(--primary-blue)] text-white py-2 px-4 rounded hover:bg-[var(--primary-blue-hover)] transition-colors"
          >
            โอนเงิน
          </button>
        </form>
      </div>
    </div>
  );
}

export default Transfer;