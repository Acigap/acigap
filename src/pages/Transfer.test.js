import React from 'react';
import { render } from '@testing-library/react';
import Transfer from './Transfer';

describe('Transfer', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Transfer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
