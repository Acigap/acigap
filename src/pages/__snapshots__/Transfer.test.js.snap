// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transfer matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="space-y-6"
  >
    <h1
      class="text-2xl font-semibold text-[var(--neutral-12)]"
    >
      โอนเงิน
    </h1>
    <div
      class="bg-[var(--neutral-1)] p-6 rounded-lg border border-[var(--border)]"
    >
      <form
        class="space-y-4"
      >
        <div>
          <label
            class="block text-sm font-medium text-[var(--neutral-11)] mb-1"
          >
            บัญชีปลายทาง
          </label>
          <input
            class="w-full p-2 rounded border border-[var(--border)] bg-[var(--neutral-2)]"
            placeholder="หมายเลขบัญชี"
            type="text"
          />
        </div>
        <div>
          <label
            class="block text-sm font-medium text-[var(--neutral-11)] mb-1"
          >
            จำนวนเงิน
          </label>
          <input
            class="w-full p-2 rounded border border-[var(--border)] bg-[var(--neutral-2)]"
            placeholder="0.00"
            type="number"
          />
        </div>
        <button
          class="w-full bg-[var(--primary-blue)] text-white py-2 px-4 rounded hover:bg-[var(--primary-blue-hover)] transition-colors"
          type="submit"
        >
          โอนเงิน
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
